### ISSUE TITLE
Implement modularized SecRel Snyk and Aqua checks

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/abd-vro#1005]
SecRel [4.0.0-rc2](https://psychic-disco-c1251ea1.pages.github.io/pipeline/release_notes/v4.0.0-rc2/):
> Introduces modularized Snyk reusable workflow. Snyk Code and SCA can now be called separately from SecRel
> Introduces modularized Aqua reusable workflow. Aqua can now be called separately from SecRel

VRO should use these for PR checks.

NOTE: we're targeting [4.0.0](https://psychic-disco-c1251ea1.pages.github.io/pipeline/release_notes/v4.0.0/) directly now (it's out of rc land)

**Comment by yoomlam** [comment]
Waiting for @sethdarragile6 to investigate Aqua alerts resulting from the using SecRel 4

**Comment by sethdarragile6** [comment]
note to @yoomlam : as a part of this ticket, I'm going to add slack notifications in as I did with the 3.2.0 version (PR #1043). It should be trivial to implement, and I figure it would save testing time while allowing the 3.2.0 notifications to merge in and do its thing in parallel

**Comment by yoomlam** [comment]
@sethdarragile6 Note that these new modularized checks will run for every PR and probably multiple times per PR. Failure notifications could clutter up the devops Slack channel and distract from more important "deploy" messages and SecRel failures on the protected branches.

**Comment by yoomlam** [comment]
Aqua checks require images. We don't publish images to GHCR for PRs. Let's say publishing images takes 5 minutes. Looking at SecRel 4 runs, Aqua checks take about 2 minutes. So a total of less than 8 minutes is acceptable for a PR check. TODO: Verify these estimated runtimes.
Since images could be published for every commit of every PR for the Aqua scan, there will be extraneous image versions. I suggest we publish these images with the `pr_` prefix so we can periodically delete them from GHCR. (This will keep the images separate from `dev_` and prod images, which we should be careful about deleting.)

@sethdarragile6 Let's check with the SecRel team to verify that this is an acceptable approach to using their modular Aqua checks.

**Comment by sethdarragile6** [comment]
Put on hold until SecRel 4.1.0 is released. [thread](https://lighthouseva.slack.com/archives/C03UA9MV1EH/p1675889032166609)

**Comment by sethdarragile6** [comment]
@dianagriffin @yoomlam  the idea was to pair on this with @greene858 this sprint, but all our pairing turned to more critical on-/off-boarding activities (which continue today w/SecRel and Dependabots processing). I recommend putting it to the backlog, as it is not mission-critical, more of a big nice-to-have in the future.

**Comment by yoomlam** [comment]
Benefit: distributes some of the work to address SecRel alert [to developers](https://dsva.slack.com/archives/C03UA9MV1EH/p1682540124893519)

**Comment by greene858** [comment]
Will take on this ticket when we have DevOps capacity...
